### ISSUE TITLE
Allow user to enable/disable line wrapping

### ISSUE THREAD

**Issue by Dimchikkk** [StaffEngineer/bevy_cosmic_edit#48]
`max_lines` doesn't work properly with long lines since line wrapping. Let's give a plugin user option to disable text wrapping.
The following code disables it:

```rs
        let mut buffer = Buffer::new(
            &mut font_system.0,
            Metrics::new(metrics.font_size, metrics.line_height).scale(metrics.scale_factor),
        );
        buffer.set_wrap(&mut font_system.0, cosmic_text::Wrap::None);
        let mut editor = Editor::new(buffer);
```

**Comment by bytemunch** [comment]
Is this solved by #58 ? I think providing the option is still a good idea but don't want to duplicate functionality

**Comment by Dimchikkk** [comment]
Not really "solved" but yeah, we can expose it later if needed.
